ElastosDIDSDK/DIDDocument.swift

```diff
@@ -262,21 +283,35 @@ public class DIDDocument {
         guard getMetadata().attachedStore else {
             throw DIDError.didStoreError("Not attached with DID store.")
         }
-        guard getMetadata().store!.containsPrivateKey(for: subject, id: ofId) else {
+        guard getMetadata().store!.containsPrivateKey(subject, ofId) else {
```

Comment:
Why want to  remove label parameter.
the original method: public func containsPrivateKey(for did: DID, id: DIDURL) -> Bool